nix/modules/scripts.nix

```diff
@@ -35,8 +35,15 @@
         set -xeuo pipefail
         trap "cd $PWD" EXIT
 
-        export VERSIONS_DIR="./versions/''${1}"
-        export DEFAULT_VERSIONS_DIR="$(nix flake metadata --no-write-lock-file --json | jq --raw-output '.locks.nodes.versions.locked.path')"
+        # Check the version of the format, otherwise processing the output might fail silently, like using jq again below
```

Comment:
Check that next time this happens we have some kind of hint to check that the change in format doesn't break the script.